sessionctx/variable/session.go

```diff
@@ -222,18 +222,13 @@ func (tc *TransactionContext) SetForUpdateTS(forUpdateTS uint64) {
 }
 
 // SetStmtFuture sets the stmtFuture .
-func (tc *TransactionContext) SetStmtFuture(future oracle.Future, cachedTS uint64) {
```

Comment:
CachedTS is used in follower-read?

Reply:
No, it is used to avoid get tso too many times when building executor. The initial implementation was to obtain tso synchronously, but later changed to asynchronous, I think that I can use reset future to make the function reentrant.